### PR TITLE
Update find ellipses

### PR DESCRIPTION
**Pull request by sturkmen72**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by asmorkalov** [comment]
@sturkmen72 Is the patch ready for review?

**Comment by sturkmen72** [comment]
@asmorkalov the patch is ready for review except "trailing whitespace" issue ( I did not understand what is the problem )

File: modules/ximgproc/samples/find_ellipses.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
```
Comment (alalek) [inline modules/ximgproc/samples/find_ellipses.py:1]: EOL (end-of-line) encoding of this file is CRLF (Windows).
Please switch to LF (UNIX style)

Some hints are here: https://stackoverflow.com/questions/27810758/how-to-replace-crlf-with-lf-in-a-single-file

**Review by alalek — APPROVED** [review]
LGTM :+1: Thank you for contribution!

**Pull request merged**